src/PitWallAcquisitionPlugin/PitWallAcquisitionPlugin/Repositories/PitWallRemoteRepository.cs

```diff
@@ -13,8 +13,8 @@ public sealed class PitWallRemoteRepository : IStagingDataRepository
          * Idea: use configuration from simhub to define this one.
          * 
          * */
-        //private const string WebApiUrl = "http://localhost:32773/api/Telemetry";
-        private const string WebApiUrl = "http://172.201.106.76:32773/api/Telemetry";
+        private const string WebApiUrl = "http://localhost:32773/api/Telemetry";
```

Comment:
issue(blocking) : hard coded address here. Will be removed on the rework on the configuration on next feature.